modules/mobile/app/controllers/mobile/v0/claims_and_appeals_controller.rb

```diff
@@ -131,6 +132,22 @@ def prepare_claims_and_appeals
         [Mobile::V0::ClaimOverviewSerializer.new(list, options), status]
       end
 
+      def log_decision_letter_sent(list)
+        decision_letters_sent = list.select(&:decision_letter_sent)
+
+        return nil if decision_letters_sent.empty?
+
+        claims_decision_letters_sent = decision_letters_sent.select { |item| item.type == 'claim' }
+        appeals_decision_letters_sent = decision_letters_sent.select { |item| item.type == 'appeal' }
+
+        Rails.logger.info('MOBILE CLAIM DECISION LETTERS SENT COUNT',
+                          user_uuid: @current_user.uuid,
+                          user_icn: @current_user.icn,
+                          claims_decision_letter_sent_count: claims_decision_letters_sent.count,
+                          appeals_decision_letter_sent_count: appeals_decision_letters_sent.count,
```

Comment:
we only show doc type 184 decision letters on the decision letter endpoint which is only associated with claims, not appeals. so seeing if appeals decision letters is set to true should be telling.